tests/munging/test_replace.py

```diff
@@ -21,6 +21,63 @@
 from tests import list_equals
 
 
+#-------------------------------------------------------------------------------
+# Frame.replace() signature
+#-------------------------------------------------------------------------------
+
+def test_replace_scalar_scalar():
+    df = dt.Frame([1, 2, 3])
+    df.replace(1, 5)
+    assert df.topython() == [[5, 2, 3]]
+
+
+def test_replace_list_scalar():
+    df = dt.Frame([1, 2, 3])
+    df.replace([1, 2, 7], 5)
+    assert df.topython() == [[5, 5, 3]]
+
+
+def test_replace_list_list():
+    df = dt.Frame([1, 2, 3])
+    df.replace([1, 2, 7], [6, 2, 5])
+    assert df.topython() == [[6, 2, 3]]
+
+
+def test_replace_emptylist():
+    df = dt.Frame([1, 2, 3])
+    df.replace([], 0)
+    assert df.topython() == [[1, 2, 3]]
+
+
+def test_replace_dict():
+    df = dt.Frame([1, 2, 3])
+    df.replace({3: 1, 1: 3})
+    assert df.topython() == [[3, 2, 1]]
+
+
+
+
+#-------------------------------------------------------------------------------
+# Replacing in bool columns
+#-------------------------------------------------------------------------------
+
+def test_replace_bool_simple():
+    df = dt.Frame([[True, False, None], [True] * 3, [False] * 3])
+    df.replace({True: False, False: True})
```

Comment:
So it does replacement once for each value, i.e. there is no way `True` will be replaced with `False` and this `False` will be again replaced with `True`?

Reply:
that's correct